tapir/statistics/views.py

```diff
@@ -206,6 +217,62 @@ def get_context_data(self, **kwargs):
         return context_data
 
 
+class AverageBasketEvolutionJsonView(LoginRequiredMixin, JSONView):
+    def get_context_data(self, **kwargs):
+        tapir_user = get_object_or_404(TapirUser, pk=(self.kwargs["pk"]))
+        logged_user = self.request.user
+        if tapir_user.pk != logged_user.pk and not logged_user.has_perm(
+            PERMISSION_COOP_MANAGE
+        ):
+            return HttpResponseForbidden("Access forbidden")
+
+        user_purchases = (
+            PurchaseBasket.objects.filter(tapir_user=tapir_user)
+            .annotate(month=TruncMonth("purchase_date"))
+            .values("month")
+            .annotate(average_gross_amount=Avg("gross_amount"))
```

Comment:
I was probably not clear enough about the number we need, sorry!
We actually want to show how much a person spends per month, so the sum of the PurchaseBasket for each month.
This is because we're trying to reach 225€/month/member for reach financial stability. 

The problem with the current number is that it's an average from the first purchase, so the longer we go, the slowest that number will change. That's why we want to show it's evolution over time.